packages/protocol/src/protocol.yml

```diff
@@ -2187,6 +2196,7 @@ Frame:
               name: string
               mimeType: string?
               buffer: binary
+        localDirectory: string?
```

Comment:
It seems like this should also include `directoryStream: WritableStream?` for remote scenarios, to make things obvious.